### ISSUE TITLE
Remove "No events in this feed yet" entry.

### ISSUE THREAD

**Issue by kevincox** [repology/repology-webapp#182]
I really don't need a notification to know that the feed is empty now. It is fine just to have an empty feed which is enough to tell me that the feed is empty.

Notably this entry:

```xml
	<entry>
		<updated>2000-01-01T00:00:00Z</updated>
		<id>tag:repology.org,2018-04-25:maintainer-repo:kevincox@kevincox.ca:nix_unstable:-1</id>
		<title>No events yet</title>
		<content>No events in this feed yet</content>
	</entry>
```

**Comment by kevincox** [comment]
Seems to come from here based on the `<content>` text:

https://github.com/repology/repology-webapp/blob/2672c9acc1861dff12743ffc8f4305dd306a367b/repologyapp/templates/repository-feed-atom.xml#L48-L53

**Comment by AMDmi3** [comment]
Sounds fair. Perhaps I've got a wrong idea that an empty feed is invalid, however it validates fine with w3c validator. However bogus `<updated>` element is still necessary.

**Comment by kevincox** [comment]
Thanks, the `<updated>` should definitely stay. Although I think the hardcoded value is actually a lie. However IDK if it is worth doing more work to make it correct.

**Comment by AMDmi3** [comment]
I don't think there's a way to make it more correct - as long as there are no events, there's no last update time (which equals to the time of last event). If we take the current time, the first event will make update time go back which I suspect is not corrent. So a fixed time earlier than any possible event should go.

**Comment by kevincox** [comment]
Well in my case there were events which got dropped off. So if the time that those events were removed was recorded it would be correct. But I don't think it is worth doing that.

**Comment by AMDmi3** [comment]
Oh, it seems that I understand the problem now - so this happens when all the events expire. In fact that dummy entry was intended for a genuinely empty feed, while a feed expiring to an emptiness is not an intended behavior.

My intention was to save (atom) clients from refetching hundreds of very old events on each and every feed update, so (unlike html feeds) events in atom feeds are limited by age. As already mentioned, emptying the feed completely was not intended, so I'll rework it to always return a few events, even if very old.

**Comment by kevincox** [comment]
I think the current situation is fine. The problem for me was that when I subscribed there were events. Eventually the events expired (which was fine) but then this "new" item appeared so I got an email! Now this unnecessary item won't reappear so there will be no problem. New items will appear again and be notified as expected.

If you want to keep a few events that sounds good too, but I don't see it as necessary.
